packages/fluentui/accessibility/src/behaviors/Tree/treeItemBehavior.ts

```diff
@@ -22,64 +22,73 @@ import treeTitleBehavior from './treeTitleBehavior';
  * Triggers 'expand' action with 'ArrowRight' on 'root', when has a closed subtree.
  * Triggers 'focusFirstChild' action with 'ArrowRight' on 'root', when has an opened subtree.
  */
-const treeItemBehavior: Accessibility<TreeItemBehaviorProps> = props => ({
```

Comment:
Does this really change something other than brackets (here and below)? If not, let's make it as it was before.